cmd/kubeapps-apis/core/plugins/v1alpha1/plugins.go

```diff
@@ -45,6 +45,10 @@ type GRPCPluginRegistrationOptions struct {
 	ConfigGetter     core.KubernetesConfigGetter
 	ClustersConfig   kube.ClustersConfig
 	PluginConfigPath string
+	// The QPS and Burst options that have been configured for any
+	// clients of the K8s API server created by plugins.
+	ClientQPS   float32
+	ClientBurst int
```

Comment:
Ah, ok, so these values are already been set by the `kube-api-qps` and `kube-api-burst` CLI flags we added in the past, aren't they?

Reply:
Yes, if we're passing them in, I didn't see a reason to use different flags for this client than those already passed in for the user client, so populate the values from the existing flags.